### ISSUE TITLE
VS2013 RC4, TypeScript 1.4, NTVS 1.0RC2: pasting a function mutilates its comment

### ISSUE THREAD

**Issue by rogierschouten** [Microsoft/TypeScript#1756]
Not sure if this should be reported here, please let me know.

Pasting the following text in a TypeScript file

```javascript
        /**
	 * Try to get resources to write some of the to-be-done archives to the database
	 */
	private _doSomeWriting(): Promise<void> {
        }
```

consistently yields

```javascript
	/**
		 ry to get resources to write some of the to-be-done archives to the database
		 
	private _doSomeWriting(): Promise<void> {
	}
```

i.e. the top comment is mutilated. Pressing Ctrl+Z once solves it.

**Comment by mhegazy** [comment]
Is that in an empty file? or did the file has something else? do you get this re-produce consistently?

**Comment by vladima** [comment]
I cannot reproduce it but symptoms look very suspicious as you describe it - formatting should never took away anything but whitespaces. Do you have anything else in this file?

**Comment by rogierschouten** [comment]
Yeah i did have other things in the file. Tomorrow at work I can see if I
can reproduce it and I'll give you a full file.

**Comment by rogierschouten** [comment]
Also, you need to have all the "Format on XXX" checkboxes checked under
Tools - Options - Text Editor - Typescript somewhere

**Comment by rogierschouten** [comment]
I've tried several times now but I can't reproduce it anymore. I was doing some heavy  refactoring and  experienced the issue in multiple files before. If I see it again I'll send the whole file.

**Comment by danquirk** [comment]
Will close this issue then. Feel free to log a new one if you see it again and can't reopen this.
